src/engines/html5/html5.js

```diff
@@ -55,6 +55,12 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @private
    */
   _canLoadMediaSourceAdapterPromise: Promise<*>;
+  /**
+   * Flag that indicates the status of the engine (playing or paused)
+   * @type {boolean}
+   * @private
+   */
+  _isPlaying: boolean = false;
```

Comment:
@odedhutzler  for playback states we can use the state manager and avoid of keep another state in the player itself.

Reply:
moved the logic to the player (and used the state manager). thanks!